tikv/kv.go

```diff
@@ -260,6 +260,34 @@ func NewPDClient(pdAddrs []string) (pd.Client, error) {
 	return pdCli, nil
 }
 
+// NewPDClientWithAPIContext returns an unwrapped pd client with API context.
+func NewPDClientWithAPIContext(pdAddrs []string, apiContext pd.APIContext) (pd.Client, error) {
```

Comment:
How about implementing `NewPDClient` with `NewPDClientWithAPIContext` by passing the APIV1 context?